pod-batch.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+kind: Pod
+metadata:
+# this value should be differernt when running script again, as each time a script is run a new Pod is created
+  name: pod1
+spec:
+  containers:
+  - name: barepod
+  # Docker image name
+    image: 172.30.223.233:5000/sample-project/batch-cgs
+    command: ["python", "groupsforsections.py"]
+    env:
+    # the below 2 are environmental variables in Open Shift
```

Comment:
I am not sure if i understood what you are trying to convey. 
If the Environmental variable name is misspelled then the script stops then and there with error message. I don't know what do you mean by `invalid values`?  If env variable name are wrong then there is no way i can access thoses values in the script.